example/src/test/java/com/capgemini/mrchecker/core/datadriven/DataDrivenInternalExampleTest.java

```diff
@@ -1,96 +1,96 @@
 package com.capgemini.mrchecker.core.datadriven;
 
-import com.capgemini.mrchecker.core.datadriven.person_example.SimpleCalc;
-import com.capgemini.mrchecker.test.core.BaseTest;
-import junitparams.JUnitParamsRunner;
-import junitparams.Parameters;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import static org.junit.Assert.assertEquals;
 
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
+import java.util.stream.Stream;
 
-import static org.junit.Assert.assertEquals;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 
-@RunWith(JUnitParamsRunner.class)
-public class DataDrivenInternalExampleTest extends BaseTest {
-	private SimpleCalc testSubject = new SimpleCalc();
-
-	/**
-	 * ---------------------------------------------------------------------------------------------------------------
-	 * --- EXAMPLE 1 -------------------------------------------------------------------------------------------------
-	 * ---------------------------------------------------------------------------------------------------------------
-	 * takes parameters from the inside the annotation
-	 **/
-	@Test
-	@Parameters({ "1, 2, 3",
-			"3, 4, 7",
-			"5, 6, 11",
-			"7, 8, 15" })
-	public void addProducesCorrectValue_usingAnnotatedParameters(final int a, final int b, final int expectedResult) {
-		assertEquals(expectedResult, testSubject.add(a, b));
-	}
+import com.capgemini.mrchecker.core.datadriven.person_example.SimpleCalc;
+import com.capgemini.mrchecker.test.core.BaseTest;
 
+public class DataDrivenInternalExampleTest extends BaseTest {
+	private final SimpleCalc testSubject = new SimpleCalc();
+	
+	// /**
+	// * ---------------------------------------------------------------------------------------------------------------
+	// * --- EXAMPLE 1 -------------------------------------------------------------------------------------------------
+	// * ---------------------------------------------------------------------------------------------------------------
+	// * NOT SUPPORTED in JUNIT5 - takes parameters from the inside the annotation
+	// **/
+	// @ParameterizedTest
+	// @ValueSource(ints ={1, 2, 3},
+	// {3, 4, 7},
+	// {5, 6, 11},
+	// {{7, 8, 15}})
+	// public void addProducesCorrectValue_usingAnnotatedValueSource(final int a, final int b, final int expectedResult)
+	// {
+	// assertEquals(expectedResult, testSubject.add(a, b));
+	// }
+	
```

Comment:
I think we should get rid of commented and unsupported tests

Reply:
Deleted